server/routes/specialRemissionRoutes.ts

```diff
@@ -162,6 +164,13 @@ export default class SpecialRemissionRoutes {
     const { nomsId } = req.params
 
     const adjustment = this.adjustmentsStoreService.getOnly(req, nomsId)
+
+    const crdString = await this.calculateReleaseDatesService.getCrdForSpecialRemission(nomsId, username)
+    const crdDate = new Date(crdString)
+    const fromDate = crdString ? new Date(crdDate.setDate(crdDate.getDate() - adjustment.days)) : new Date()
```

Comment:
Has this come from policy? Using the sentence date rather than a number of days before release would be much easier!

If this really is what policy want, it might be better suited being within `calculate-release-dates-api` where it can be tested against some real calculations.

Reply:
It has, policy don't need a date for special remission going forward but this was their suggestion for what to use for backwards compatibility with Nomis (Syscon just suggested using the date the special remission was created which we default to)